2006Scape Server/src/main/java/com/rs2/game/players/Player.java

```diff
@@ -144,6 +144,24 @@ public abstract class Player {
 	private GateHandler gateHandler = new GateHandler();
 	private SingleGates singleGates = new SingleGates();
 	private DoubleGates doubleGates = new DoubleGates();
+	
+	private Map<Integer, Integer> npcKillCounts = new HashMap<>();
+	public boolean displayBossKcMessages = true;
```

Comment:
this should be false by default

Reply:
In that case it should be a client setting and a flag.

Reply:
Oh wait it's the server nevermind me I'm tired lol

Reply:
I already made it a flag, cool, I'll disable it by default.